### ISSUE TITLE
Create `CalibrationInformation ` instances and upload to Mongo

### ISSUE THREAD

**Issue by brynnz22** [microbiomedata/issues#750]
In order to submit change sheets for the current `MassSpectrometry` instances in Mongo, we will need to include mapping any `has_calibration` info for instances that it applies to. This has not yet been started and should be added to a sprint when ready. 

- [x] Track down calibration `DataObject`s in Mongo
- [x] Make initial `CalibrationInformation` spreadsheet for the existing  calibration `DataObject`s 
- [x] Manually add name and description for `CalibrationInformation`
- [x] Mint ids for each instance @brynnz22 
- [x] Convert to json and upload to Berkeley Mongo. @brynnz22 

Completing the above steps will unblock these outstanding issues
- https://github.com/microbiomedata/nmdc-schema/issues/2139 @kheal

**Comment by kheal** [comment]
@brynnz22  looks great.  Just for clarity, there are existing DataObjects in Mongo for the existing (GC-MS) metabolomics data.  Those will be straightforward to map.  

Yuri also wanted to add CalibrationInformation for the NOM MassSpectrometry instances.  They will all share the same DataObject, but it will be a different mapping.

**Comment by ssarrafan** [comment]
Mentioned a lot so assuming it's active. Moving to next sprint.

**Comment by brynnz22** [comment]
Blocked until Berkeley submitter permissions are updated. @eecavanna

**Comment by eecavanna** [comment]
I'll be OOO until tomorrow (back in the office on Wednesday).

I'm not familiar with the "Berkeley submitter permissions." The Berkeley Mongo database was initialized from a production dump (which was then run through the Berkeley migrators). It sounds to me like there is a permission-related issue that is blocking progress on this ticket (and some other tickets), but I don't know what the issue is. If it is something that can be resolved in < 15 mins or so, I can do that today.

**Comment by brynnz22** [comment]
@eecavanna I was waiting to message you until after your vacation so you wouldn't feel the need to respond :) When I go to submit a json (and I'm gonna guess a change sheet as well) in the Berkeley runtime - it gives a 200 response code, but when I check dagster it says `Exception: Getting token failed: {'detail': 'Incorrect client_id or client_secret'}`. @sujaypatil96 tried it out too and same thing happens to him.

**Comment by eecavanna** [comment]
Ah, gotcha. Thank you, @brynnz22! I appreciate that! Since I don't immediately know how to fix that issue, I will leave it for tomorrow.

**Comment by aclum** [comment]
@brynnz22  this should be okay to try again. I just tested Donny's fix and was able to add records to berkeley mongo with json:submit.

**Comment by brynnz22** [comment]
Yep it works, I'll move this off of the sprint since we will have to officially submit one Berkeley becomes production (since Berkeley Mongo is not replacing production Mongo).

**Comment by brynnz22** [comment]
Final json for this (includes two `data_object_set` instances for the nom calibrations that don't yet exist in Mongo, but needed for NOM ingest script:[calib_config.json](https://github.com/user-attachments/files/17103046/calib_config.json)

**Comment by bmeluch** [comment]
JSON submitted through /metadata/json:submit in dev and prod

**Comment by brynnz22** [comment]
Thanks @bmeluch

**Comment by kheal** [comment]
Closing this - just the migrator left!

**Comment by brynnz22** [comment]
I thought the migrator was added as a checkbox to this. DId you remove it? That's why I didn't close.

**Comment by kheal** [comment]
I did remove that checkbox it since it is covered by a separate issue (the one noted as previously blocked before) and felt like it's own thing. @brynnz22
